src/main/java/de/adesso/kicker/email/EmailService.java

```diff
@@ -5,39 +5,44 @@
 import de.adesso.kicker.user.persistence.User;
 import lombok.RequiredArgsConstructor;
 import org.springframework.context.event.EventListener;
-import org.springframework.mail.SimpleMailMessage;
+import org.springframework.context.i18n.LocaleContextHolder;
 import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.mail.javamail.MimeMessagePreparator;
 import org.springframework.stereotype.Service;
 
-import java.util.ArrayList;
-import java.util.Objects;
+import java.text.MessageFormat;
+import java.util.*;
 
 @Service
 @RequiredArgsConstructor
 public class EmailService {
 
+    private final JavaMailSender mailSender;
+
+    private final EmailMessageBuilder emailMessageBuilder;
+
     static final String ACCEPT_URL = "http://localhost/notifications/accept/";
```

Comment:
The URL's can be part of the template

Reply:
> > I haven't implemented tests yet.
> 
> I read that and this won't be merged unless there are tests.

I thought that would be obvious, that it's not getting merged without tests